torch/_dynamo/variables/misc.py

```diff
@@ -570,12 +570,16 @@ def call_function(
         if is_original_tensor_torch_function:
             # Instead of tracing inside torch.Tensor.__torch_function__,
             # record the `call_function` or `call_method` call into the graph.
-            from . import TorchVariable
+            from . import ConstantVariable, ConstDictVariable, TorchVariable
 
             original_torch_or_getattr_variable = args[0]
             new_args = args[2].items
-            new_kwargs = args[3].items
-            options = VariableTracker.propagate(self, new_args, new_kwargs.values())
+            if not isinstance(args[3], ConstantVariable):
+                new_kwargs = args[3].items
```

Comment:
how do you know `.items` exists?  Are you assuming `not isinstance(args[3], ConstantVariable)` implies some specific type?

Reply:
Yeah this kwargs - I needed this code that I can pass a test, in the next PR I'm going to remove this entirely because we will properly trace the default `__torch_function__` impl